test/vizkit3d.cpp

```diff
@@ -0,0 +1,46 @@
+#include "../src/Vizkit3DWidget.hpp"
```

Comment:
this is just for testing purposes but not unit tests.. maybe should be removed, maintainer should decide